package.json

```diff
@@ -374,14 +377,15 @@
     "**/kind-of": "^6.0.3",
     "**/glob-parent": "^5.1.2",
     "**/ansi-regex": "^5.0.1",
-    "**/yargs-parser": "13.1.2",
     "@babel/runtime": "^7.15.4",
-    "punycode": "1.4.1",
     "**/cached-path-relative": "1.1.0",
-    "**/moment": "^2.29.2",
```

Comment:
Was this removed on purpose? I assume we still need it in there, no?

Reply:
```
"**/yargs-parser": "13.1.2"
"punycode": "1.4.1"
"**/moment": "^2.29.2"
```
We removed these because otherwise Yarn would attempt to alter the .lockfile (even when the were no dendencu changes), in turn blocking CI

Reply:
without removing these listings in the `resolutions` field `yarn install --frozen-lockfile` throws an error: `error Your lockfile needs to be updated, but yarn was run with '--frozen-lockfile'.`

Reply:
Right, but I'm pretty sure these are in there for a reason. Usually security issues. Please confirm before merging

Reply:
This is the PR from @jhonnyoddball that set the moment resolution: https://github.com/department-of-veterans-affairs/vets-website/pull/20768

Webpack 5 upgrade brought in punycode https://github.com/department-of-veterans-affairs/vets-website/pull/19403

Reply:
Here's more background on the punycode/IE issue: https://github.com/mathiasbynens/punycode.js/issues/86

Reply:
Yeah, it was there because of security vulnerability

Reply:
We will need to find a work around it

Reply:
With the latest changes in this PR, looking at the `yarn.lock`, I see:
```
"moment@>= 2.9.0", moment@^2.22.1:
  version "2.29.4"
```
and
```
punycode@1.3.2, punycode@1.4.1, punycode@^1.3.2, punycode@^2.1.0, punycode@^2.1.1:
  version "1.4.1"
```
We should be fine with resolving to those versions. I would only be concerned if I saw yarn.lock changing to resolve to an older version of moment (security issues) or a newer version of punycode (IE issues)